build-mfem/action.yml

```diff
@@ -100,30 +112,50 @@ runs:
         echo "--- Configure mfem ---"
         cd ${{ inputs.mfem-dir }};
         if [[ ${{ inputs.build-system }} == 'cmake' ]]; then
+            if [[ ${{ inputs.os }} == "windows-2022" ]]; then
+              toolchain_file="${VCPKG_INSTALLATION_ROOT}\\scripts\\buildsystems\\vcpkg.cmake"
+              vcpkg_triplet="x64-windows-static"
+            fi
             mkdir build && cd build;
             cmake \
+              -D CMAKE_TOOLCHAIN_FILE:STRING=${toolchain_file} \
+              -D VCPKG_TARGET_TRIPLET=${vcpkg_triplet} \
               -D CMAKE_BUILD_TYPE=${BUILD_TYPE} \
               -D MFEM_USE_MPI=${MPI} \
               -D MFEM_MPI_NP=2 \
-              -D HYPRE_DIR=../../hypre/src/hypre \
+              -D HYPRE_DIR=../../hypre/install \
+              ${{ inputs.config-options }} \
               ..
         else
-            make config MFEM_USE_MPI=${MPI} MFEM_MPI_NP=2 MFEM_DEBUG=${DEBUG} CPPFLAGS="${CPPFLAGS}";
+            make config \
+              MFEM_USE_MPI=${MPI} \
+              MFEM_MPI_NP=2 \
+              MFEM_DEBUG=${DEBUG} \
+              CPPFLAGS="${CPPFLAGS}" \
+              HYPRE_DIR=@MFEM_DIR@/../hypre/install \
+              ${{ inputs.config-options }}
         fi
       shell: bash
 
     - name: build mfem
       run: |
         echo "------------------"
         echo "--- Build mfem ---"
+        BUILD_TYPE="Release"
+        [[ ${{ inputs.target }} == 'dbg' ]] && BUILD_TYPE="Debug";
         cd ${{ inputs.mfem-dir }}
         if [[ ${{ inputs.build-system }} == 'cmake' ]]; then
+            CMAKE_TARGET="exec"
+            [[ ${{ inputs.library-only }} == 'true' ]] && CMAKE_TARGET="mfem";
             cd build
-            make -j3 mfem examples
-            cd tests/unit
-            make -j3
+            cmake --build . --parallel 3 --target ${CMAKE_TARGET} --config "${BUILD_TYPE}"
```

Comment:
👍🏻

Reply:
Don’t we need to build the examples? Are the examples built?

Reply:
Examples and tests are all included in the `exec` cmake target.

Reply:
👍🏻